app-web/plugins/gatsby-source-github-all/utils/plugins.js

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 
 Created by Patrick Simonian
 */
-
+const shortid = require('shortid'); // eslint-disable-line
```

Comment:
I think you can wrap all of these in` /* eslint-disable */` and `/* eslint-enable */`.

Reply:
i'll check that out